index.d.ts

```diff
@@ -0,0 +1,9 @@
+import { Schema } from "mongoose";
+declare namespace wrapCallback {
```

Comment:
What is this `declare namespace` call about? Shouldn't we just have `interface WrapCallbackOptions {}` and just use that? And `export default function wrapCallback()`?

Reply:
I'm not sure why, but without the namespace I get the following error.

```
test.ts:1:31 - error TS2497: This module can only be referenced with ECMAScript imports/exports by turning on the 'esModuleInterop' flag and referencing its default export.

1 import * as wrapCallback from '.';
```

test.ts is:
```
import * as wrapCallback from '.';

wrapCallback({} as any);
```